app/views/decidim/analytics/admin/analytics/index.html.erb

```diff
@@ -1,18 +1,75 @@
-<div class="card" id="analytics">
-  <div class="card-divider">
-    <h2 class="card-title">
-        <%= t "decidim.admin.title.analytics" %>
-    </h2>
-  </div>
-  <div class="card-section">
-    <iframe src="<%= @server_address %>/index.php?module=Widgetize&action=iframe&moduleToWidgetize=Dashboard&actionToWidgetize=index&idSite=<%= @site_id %>&period=week&date=yesterday&token_auth=<%= @token_auth %>"
-        frameborder="0"
-        marginheight="0"
-        marginwidth="0"
-        width="100%"
-        height="4200px">
-    </iframe>
+<% add_decidim_page_title(t("decidim.analytics.admin.title.analytics")) %>
+
+<div class="row">
+  <div class="columns">
+    <div class="card" id="analytics">
+      <div class="card-divider">
+        <h2 class="card-title">
+          <%= icon "bar-chart-2-line", class: "icon--before" %>
+          <%= t("decidim.analytics.admin.title.analytics") %>
+        </h2>
+      </div>
+
+      <div class="card-section">
+        <% if @server_address.present? && @site_id.present? && @token_auth.present? %>
+          <div class="analytics-iframe-container">
+            <iframe
+              src="<%= matomo_dashboard_url %>"
+              frameborder="0"
+              marginheight="0"
+              marginwidth="0"
+              width="100%"
+              height="4200px"
+              title="<%= t("decidim.analytics.admin.iframe_title") %>">
+            </iframe>
+          </div>
+        <% else %>
+          <div class="callout warning">
+            <h5><%= t("decidim.analytics.admin.configuration_missing") %></h5>
+            <p><%= t("decidim.analytics.admin.configuration_help") %></p>
+
+            <h6><%= t("decidim.analytics.admin.configuration_instructions") %></h6>
+            <pre><code># config/credentials.yml.enc (recomendado)
```

Comment:
As I told you before, we don't use credentials. We should have the module configuration so we can do something like `Decidim::Analytics.matomo[:server_address]` to access it. And this configuration would take the values from env vars like `DECIDIM_ANALYTICS_MATOMO_SERVER_ADDRESS`. Check it out how decidim handle the configuration of the maps for example. You can configure it by only defining env variables.